pkg/volume/glusterfs/glusterfs.go

```diff
@@ -255,8 +260,19 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	for hostIP := range addr {
 		errs = b.mounter.Mount(hostIP+":"+b.path, dir, "glusterfs", options)
 		if errs == nil {
+			glog.Infof("glusterfs: successfully mounted %s", dir)
 			return nil
 		}
 	}
+
+	// Failed mount scenario.
+	// Since gluster does not return eror text
+	// it all goes in a log file, we will read the log file
+	logerror := readGlusterLog(log, b.pod.Name)
+	if logerror != nil {
+		glog.Errorf("glusterfs: mount failed: %v", logerror)
+		return fmt.Errorf("glusterfs: mount failed: %v", logerror)
```

Comment:
please also add a line between `errs` and `logerror` to tell end user the second error is from gluster,  something like

```go
return fmt.Errorf("glusterfs: mount failed: %v\n use the following information with glusterfs admin \n %v", errs,  logerror)
```